### PR TITLE
Tarronlane

### PR DESCRIPTION
**Pull request by tarronlane**

### CODE REVIEW

File: functions/graphql/index.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ function configureServer() {
       // CENTER
       // const token = "123456789012345678901234567890123456"; // Must be a 36 long string that matches the apiToken object on a center
       // DIRECTOR
-      // const token = await User.getToken('directora@example.com', 'Test1234'); // user id: ykR3TSwRYAXcvqgYGaNR2G3Px2w2
+      const token = await User.getToken('directora@example.com', 'Test1234'); // user id: ykR3TSwRYAXcvqgYGaNR2G3Px2w2
```
Comment (alexanderson1993) [inline functions/graphql/index.js]: Make sure this is commented out so it doesn't get broken for me.
```suggestion
      // const token = await User.getToken('directora@example.com', 'Test1234'); // user id: ykR3TSwRYAXcvqgYGaNR2G3Px2w2
```

File: functions/graphql/index.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ function configureServer() {
     uploads: false,
     context: async ({ req }) => {
       // PRODUCTION VERSION
-      const token = (req.headers.authorization || "").replace("Bearer ", "");
+      // const token = (req.headers.authorization || "").replace("Bearer ", "");
```
Comment (alexanderson1993) [inline functions/graphql/index.js]: Make sure this is uncommented so it doesn't get broken for me.
```suggestion
      const token = (req.headers.authorization || "").replace("Bearer ", "");
```

File: functions/graphql/schema/flightRecord.js [hunk 3]
```diff
@@ -63,7 +63,7 @@ module.exports.schema = gql`
       date: Date
       flightTypeId: ID
       simulators: [FlightSimulatorInput]
-    ): FlightRecord @auth(requires: [director])
+    ): Boolean @auth(requires: [director])
```
Comment (alexanderson1993) [inline functions/graphql/schema/flightRecord.js]: Why does this need to be a Boolean?

Comment (tarronlane) [reply functions/graphql/schema/flightRecord.js]: Just to show if it was updated successfully or not, but I'm not attached to it being boolean if you want a more useful result.

Comment (alexanderson1993) [reply functions/graphql/schema/flightRecord.js]: Having it be a Flight Record that is returned (and throwing an error if it fails) makes it easier for me to update the UI with the results of the mutation.
```suggestion
    ): FlightRecord @auth(requires: [director])
```

**Pull request merged**